src/Symfony/Component/HttpFoundation/Tests/File/FileTest.php

```diff
@@ -13,7 +13,7 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\HttpFoundation\File\File;
-use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Symfony\Component\Mime\MimeTypes;
 
 class FileTest extends TestCase
```

Comment:
now that now mock guesser are used, we should add the `@requires extension fileinfo` annotation on this class